php/libraries/NDB_BVL_Battery.class.inc

```diff
@@ -352,7 +352,32 @@ class NDB_BVL_Battery
         // return all the data selected
         return $rows;
     } // end getBatteryVerbose()
-
+    /**
+     * Checks to see if a table contains a specified column
+     *
+     * @param string $table_name The table to check for a column.
+     * @param string $column     The column name to check the table for.
+     *
+     * @return boolean true if the table has at least one NULL in the column
+     * @note   This should really be in the Database.class.inc file... -anyhowstep
```

Comment:
```suggestion
```
This comment should be removed. I think it's also telling the truth, however. I think this method should belong to `php/libraries/Database.inc` rather than `php/libraries/NDB_BVL_Battery.class.inc`. Can you explain why it was moved here? Just curious.

Reply:
Hi @johnsaigle, I moved the code to NDB_BVL_Battery because this seems the only class to use it other than Utility test.

Reply:
Oh I see. That makes sense. Thanks!